### ISSUE TITLE
Breaks Medizinfuchs.de

### ISSUE THREAD

**Issue by kennymc-c** [oblador/hush#166]
Hush breaks Medizinfuchs.de. The site is completely empty even if you have accepted any of the cookie options before when the plugin was disabled.

**Comment by MinziMinzz** [comment]
Yeah, medizinfuchs.de breaks, because there is a class cookie-hint on the body, that's why the whole page doesn't load.
Hush blocks every element, that has words like cookie in it.
